common4j/src/main/com/microsoft/identity/common/java/constants/FidoConstants.kt

```diff
@@ -36,5 +36,33 @@ class FidoConstants {
          * Extra query parameter value to declare WebAuthn capability for a host app.
          */
         const val WEBAUTHN_QUERY_PARAMETER_VALUE = "1"
+
+        /**
+         * Header name to signal that the custom passkey protocol should be used.
+         */
+        const val PASSKEY_PROTOCOL_HEADER = "x-ms-PassKeyAuth"
+
+        /**
+         * Version of the passkey protocol that we want to use.
+         */
+        const val PASSKEY_PROTOCOL_VERSION = "1.0"
+
+        /**
+         * Key types that we support with the passkey protocol.
+         * Possible values: securitykey, passkey, ngc
```

Comment:
Are these values defined as consts somewhere?

Reply:
Added them to the same file: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/2181/commits/8a90970d55796d97a386628c3bd025d82e46f23f